### ISSUE TITLE
Data Sanitization

### ISSUE THREAD

**Issue by OR13** [w3c-ccg/universal-wallet-interop-spec#34]
It should be possible to "show your wallet contents" to another party, without loosing any controlling authority that comes from any secret (private) information in wallet contents.

Doing so will of course allow that party to track you everywhere you use any public components of the content, but that party should not be able to impersonate you, or decrypt messages intended for you.

**Comment by TelegramSam** [comment]
Can you explain the purpose behind such a feature?

**Comment by OR13** [comment]
@TelegramSam yep, essentially "keypairs" have private keys, but showing them is dangerous... what if you could automatically ask for data models that are "safe to share" in that they reveal correlation, but don't lead to compromise.

**Comment by OR13** [comment]
JSON Schema for type matching would help identify content that should be filtered properly.
